src/shell_minimal.js

```diff
@@ -4,18 +4,20 @@
  * SPDX-License-Identifier: MIT
  */
 
-#if USE_CLOSURE_COMPILER
+#if MODULARIZE && EXPORT_NAME != 'Module'
+// We have a lot internal/library code that depends on `Module` so
+// we create an alias for EXPORT_NAME (which is passed in as arg0).
+var Module = {{{ EXPORT_NAME }}};
+#elif ENVIRONMENT_MAY_BE_NODE || ENVIRONMENT_MAY_BE_SHELL
+// When running on the web we expect Module to be defined externally, in the
+// HTML.  Otherwise we must define it here before its first use
+var Module = Module || {};
```

Comment:
If I'm reading the diff right, previously in non-MODULARIZE mode on Node.js this would use existing `{{ EXPORT_NAME }}` variable for config, but now it looks only at the `Module`.

Is this still a non-functional change? I suppose not?